python/pen_checker_cdc.py

```diff
@@ -278,11 +306,18 @@ def process_pbp(isolate,protein_fasta,tpd_start,tpd_end,tpd_lab,results_csv,k):
                                                                 correct_length,
                                                                 gene_rower)
                 else:
-                    hmm_search_for_gene(fasta_file, files_for_input.pbp)
-                    exit(0)
+
+                    current_res, current_isolate = hmm_search_for_gene(fasta_file, files_for_input.pbp, aa_dir_name, files_for_input.data_dir)
+                    gene_id.append(current_res)
+                    isolate_name.append(current_isolate)
+                    skip = True
+                    continue
```

Comment:
Getting it to skip through remaining pbp workflow if co-trimoxazole resistance and return resistance and isolate name